src/com/lingoport/demo/java/ExampleMain.java

```diff
@@ -52,6 +52,7 @@ public static void main(String[] args){
         frame.pack();
         frame.setVisible(true);
         String strTitle = "Example Address Book -- " + locale;
+        String strTitle3 = "Example Addresss Book -- " + strTitle; 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "strTitle3". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "strTitle3" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)